DorisProject/src/app/pages/modal/modal.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal.component.html',
+  styleUrls: ['./modal.component.scss']
+})
+export class ModalComponent implements OnInit {
+
+
+  constructor(public dialogRef: MatDialogRef<ModalComponent>,
+              private  router: Router,
```

Comment:
Codacy has a fix for the issue: [Replace `··············private··` with `····private·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
    private router: Router,
```